source/endpoints/urls.md.erb

```diff
@@ -0,0 +1,755 @@
+      "1": 1,
+      "-1": 0
+    },
+    "url_group_id": 36241,
+    "places_info": null,
+    "places_match": null,
+    "places_keyword": null,
+    "places_image_title": null,
+    "custom_name": "",
+    "errors": {},
+    "youtube_video_id": null,
+    "youtube_channel": null,
+    "owner_name": "JURGLES",
+    "competitors": [
+      {
+        "id": 123,
+        "url": "example-competitor.com"
+      },
+      ...
+    ],
+    "google_indexed_change": 0,
+    "report_count": 0,
+    "google_analytics_profile_id": 105135374,
+    "country_code": null,
+    "language_code": null,
+    "keyword_suggestions_updated_at": "2019-07-17T06:40:23.139Z",
+    "google_analytics_error": null,
+    "search_console_url": null,
+    "backlinks_enabled": true,
+    "backlinks_status": {
+      "backlinks_total": 0,
+      "last_ahrefs_fill_try": "2019-07-20 20:15:10 +0000",
+      "ahrefs_is_currently_fetching": false,
+      "ahrefs_initial_fetching_done_at": "2017-11-22 08:37:10 +0000",
+      "ahrefs_recurring_fetching_done_at": "2019-07-20 20:15:31 +0000"
+    },
+    "backlinks_count": 4,
+    "google_analytics_updated_at": "2019-07-21T13:10:19.973Z",
+    "backlinks_fetching_enabled": true,
+    "include_local_pack_in_main_position": true,
+    "include_places_image_in_main_position": true,
+    "include_featured_snippet_in_main_position": true,
+    "include_knowledge_panel_in_main_position": true,
+    "crawling_session_url_params_enabled": true,
+    "crawling_session_follow_nofollow_links": true,
+    "crawling_session_path_ignore_pattern": null,
+    "next_site_audit_crawl_at": null,
+    "site_audit_interval": null,
+    "backlinks_settings": {
+      "ignored_domains": []
+    },
+    "seo_metrics": {
+      "moz_rank": 0.1,
+      "moz_domain_authority": 1,
+      "moz_page_authority": 1,
+      "majestic_trust_flow": 0,
+      "majestic_citation_flow": 0,
+      "ahrefs_rank": 0,
+      "ahrefs_domain_rating": 0
+    },
+    "seo_metrics_changes": {
+      "moz_rank_changed_at": "2018-11-05",
+      "moz_rank_change": -0.2,
+      "moz_domain_authority_changed_at": "2019-03-08",
+      "moz_domain_authority_change": -1,
+      "moz_page_authority_changed_at": "2018-11-05",
+      "moz_page_authority_change": -2,
+      "majestic_citation_flow_changed_at": "2019-04-04",
+      "majestic_citation_flow_change": -2,
+      "ahrefs_rank_changed_at": "2018-12-18",
+      "ahrefs_rank_change": -3
+    },
+    "match_subdomains": false,
+    "match_nested_urls": true
+  },
+  ...
+]
+```
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/urls?access_token=ACCESS_TOKEN`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+group_id  | Only return URLs belonging to this group (optional)
+
+
+## Get a URL
+
+```shell
+curl 'https://api.nightwatch.io/api/v1/urls/31234?access_token=ACCESS_TOKEN' \
+  -H 'Content-Type: application/json'
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
```

Comment:
Let's not paste this huge response for every endpoint. A couple of fields would suffice.